cfgov/templates/wagtailadmin/css/table-block.css

```
@@ -1,3 +1,23 @@
+.Draftail-Editor__wrapper {
+ height: auto !important;
+ width: 800px !important;
+ max-width: 800px !important;
+}
+
+.Draftail-Editor .DraftEditor-root {
+    padding: 0.75em;
+}
+
+.handsontableInputHolder {
+  left: 0 !important;
+}
+
```

```suggestion

.handsontableInputHolder .Draftail-Editor__wrapper {
  overflow-y: visible !important;
}

```

That should fix the link editor popover from getting clipped. I'm not totally sure why `overflow-y: hidden` is set on the `.Draftail-Editor__wrapper` element in the first place, but overriding it doesn't seem to break anything as far as I could tell.